installer/install.sh

```diff
@@ -263,11 +293,26 @@ function run_installer() {
   fi
 }
 
+# Run all scripts to install after the checks
+function run_installer() {
+  echo -e "\nThe Mobile installer requires valid DockerHub credentials
+  to communicate with the DockerHub API. If you enter invalid credentials or then
+  Mobile Services will not be available in the Service Catalog.\n"
+
+  read_docker_hub_credentials
+  read_docker_hub_tag
+  read_docker_hub_organization
+  read_cluster_ip
+  read_wildcard_dns_host
+  run_ansible_tasks
```

Comment:
It action required here is encapsulated which makes it more readable and easier than before to work with.